### ISSUE TITLE
Crash after reloading Front Page

### ISSUE THREAD

**Issue by SpoiledSoul** [ccrama/Slide#316]
Scrolled to the bottom of the Front Page, and when new posts wouldn't load, I decided to scroll back to the top and refresh the page. Loading bubble appeared and then Slide crashed.

Debug info:
 OS Version: 3.10.61-5496010(G925PVPU3BOI1)
 OS API Level: 22
 Device: zeroltespr
 Model (and Product): SM-G925P (zeroltespr)
 RELEASE: 5.1.1
 BRAND: samsung
 DISPLAY: LMY47X.G925PVPU3BOI1
 CPU_ABI: arm64-v8a
 CPU_ABI2: 
 HARDWARE: samsungexynos7420
 MANUFACTURER: samsung
 SLIDE VERSION: 72

Stacktrace: 

```java.lang.IndexOutOfBoundsException: Inconsistency detected. Invalid item position 0(offset:47).state:47
	at android.support.v7.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:4401)
	at android.support.v7.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:4359)
	at android.support.v7.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:1961)
	at android.support.v7.widget.LinearLayoutManager.layoutChunk(LinearLayoutManager.java:1370)
	at android.support.v7.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1333)
	at android.support.v7.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:562)
	at android.support.v7.widget.RecyclerView.dispatchLayout(RecyclerView.java:2864)
	at android.support.v7.widget.RecyclerView.onLayout(RecyclerView.java:3071)
	at android.view.View.layout(View.java:16922)
	at android.view.ViewGroup.layout(ViewGroup.java:5405)
	at android.support.v4.widget.SwipeRefreshLayout.onLayout(SwipeRefreshLayout.java:581)
	at android.view.View.layout(View.java:16922)
	at android.view.ViewGroup.layout(ViewGroup.java:5405)
	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1702)
	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1556)
	at android.widget.LinearLayout.onLayout(LinearLayout.java:1465)
	at android.view.View.layout(View.java:16922)
	at android.view.ViewGroup.layout(ViewGroup.java:5405)
	at android.support.v4.view.ViewPager.onLayout(ViewPager.java:1627)
	at android.view.View.layout(View.java:16922)
	at android.view.ViewGroup.layout(ViewGroup.java:5405)
	at android.support.design.widget.CoordinatorLayout.layoutChild(CoordinatorLayout.java:1034)
	at android.support.design.widget.CoordinatorLayout.onLayoutChild(CoordinatorLayout.java:744)
	at android.support.design.widget.ViewOffsetBehavior.onLayoutChild(ViewOffsetBehavior.java:42)
	at android.support.design.widget.AppBarLayout$ScrollingViewBehavior.onLayoutChild(AppBarLayout.java:1180)
	at android.support.design.widget.CoordinatorLayout.onLayout(CoordinatorLayout.java:757)
	at android.view.View.layout(View.java:16922)
	at android.view.ViewGroup.layout(ViewGroup.java:5405)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:579)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:514)
	at android.view.View.layout(View.java:16922)
	at android.view.ViewGroup.layout(ViewGroup.java:5405)
	at android.support.v4.widget.DrawerLayout.onLayout(DrawerLayout.java:1043)
	at android.view.View.layout(View.java:16922)
	at android.view.ViewGroup.layout(ViewGroup.java:5405)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:579)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:514)
	at android.view.View.layout(View.java:16922)
	at android.view.ViewGroup.layout(ViewGroup.java:5405)
	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1702)
	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1556)
	at android.widget.LinearLayout.onLayout(LinearLayout.java:1465)
	at android.view.View.layout(View.java:16922)
	at android.view.ViewGroup.layout(ViewGroup.java:5405)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:579)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:514)
	at android.view.View.layout(View.java:16922)
	at android.view.ViewGroup.layout(ViewGroup.java:5405)
	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1702)
	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1556)
	at android.widget.LinearLayout.onLayout(LinearLayout.java:1465)
	at android.view.View.layout(View.java:16922)
	at android.view.ViewGroup.layout(ViewGroup.java:5405)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:579)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:514)
	at android.view.View.layout(View.java:16922)
	at android.view.ViewGroup.layout(ViewGroup.java:5405)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:579)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:514)
	at android.view.View.layout(View.java:16922)
	at android.view.ViewGroup.layout(ViewGroup.java:5405)
	at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:2413)
	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2122)
	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1264)
	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6944)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:777)
	at android.view.Choreographer.doCallbacks(Choreographer.java:590)
	at android.view.Choreographer.doFrame(Choreographer.java:560)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:763)
	at android.os.Handler.handleCallback(Handler.java:739)
	at android.os.Handler.dispatchMessage(Handler.java:95)
	at android.os.Looper.loop(Looper.java:145)
	at android.app.ActivityThread.main(ActivityThread.java:6837)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1404)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1199)
```

**Comment by Alexendoo** [comment]
I can reproduce this one also

**Comment by ccrama** [comment]
Fixed 69c0ef81c58fe5bd14530264604e9f9ec0e36bca
